Kudu.Services/Deployment/PushDeploymentController.cs

```diff
@@ -222,6 +237,50 @@ public class PushDeploymentController : ApiController
             repository.Commit(zipDeploymentInfo.Message, zipDeploymentInfo.Author, zipDeploymentInfo.AuthorEmail);
         }
 
+        private async Task LocalZipFetchAndDeploy(IRepository repository, DeploymentInfoBase deploymentInfo, ILogger logger, ITracer tracer)
+        {
+            var zipDeploymentInfo = (ZipDeploymentInfo)deploymentInfo;
+
+            // For this kind of deployment, RepositoryUrl is a local path to the temp zip.
+            var sourceZipFile = zipDeploymentInfo.RepositoryUrl;
+            // the name of the deployed file in D:\home\data\SitePackages\{name}.zip is the 
+            // timestamp in the format yyyMMddHHmmss. 
+            var fileName = $"{DateTime.UtcNow.ToString("yyyyMMddHHmmss")}.zip";
```

Comment:
as the comment is saying. the name of the deployed zip is a timestamp. Any other preference?

Reply:
I think that's fine.